### PR TITLE
Manually start challenges

### PR DESCRIPTION
**Pull request by vanruch**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] index.js exports updated
- [x] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [x] Version updated

### CODE REVIEW

File: test/actions/challenges_actions.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+      challengesActions = new ChallengesActions(mockChallengesWrapper, mockFeesWrapper, mockShelteringWrapper, mockBlockchainStateWrapper);
+    });
+
+    it('calls start method of wrapper, returns correct data', async () => {
+      expect(await challengesActions.startChallenge(exampleSheltererId, exampleBundleId)).to.deep.equal({
+        ...exampleTxResult,
+        challengeId: exampleChallengeId
+      });
+      expect(mockChallengesWrapper.start).to.be.calledOnceWith(exampleSheltererId, exampleBundleId);
```
Comment (krzysztof-jelski) [inline test/actions/challenges_actions.js]: Line not needed, as we know that `exampleTxResult` can come only. from `mockChallengesWrapper`

File: test/wrappers/challenges_wrapper.js [hunk 2]
```diff
@@ -159,6 +159,93 @@ describe('Challenges Wrapper', () => {
...
+      startChallengeSendStub = sinon.stub();
+      startChallengeStub.returns({
+        send: startChallengeSendStub
+      });
+      contractMock = {
+        methods: {
+          start: startChallengeStub
+        }
+      };
+      challengesWrapper = new ChallengesWrapper({}, {}, defaultAddress, true);
```
Comment (krzysztof-jelski) [inline test/wrappers/challenges_wrapper.js]:
```suggestion
      challengesWrapper = new ChallengesWrapper({}, {}, defaultAddress);
```

I'd prefer to rely on the default value, especially because we plan to get rid of this flag altogether.

File: test/actions/challenges_actions.js (hunk 1 above)
Comment (vanruch) [reply test/actions/challenges_actions.js]: Checking that it was called once is still useful, while checking params is indeed not necessary

File: src/actions/fees_actions.js [hunk 3]
```diff
@@ -34,4 +34,8 @@ export default class FeesActions {
   async feeForUpload(storagePeriods) {
     return this.feesWrapper.feeForUpload(storagePeriods);
   }
+
+  async feeForChallenge(storagePeriods) {
+    return this.feesWrapper.feeForUpload(storagePeriods);
```
Comment (krzysztof-jelski) [inline src/actions/fees_actions.js]: Copy paste error

File: src/tasks/challenges.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  }
+
+  validateAddress(address) {
+    if (!utils.isAddress(address)) {
+      throw `Invalid address: ${address}`;
+    }
+  }
+
+  validateId(validatedId) {
+    if (!/^0x[0-9a-f]{64}$/i.exec(validatedId)) {
```
[the remark this answers is not in the document]
Comment (vanruch) [reply src/tasks/challenges.js]: We don't have any utils for validations. Those 2 lines above are web3.utils

File: src/actions/challenges_actions.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      challengeId
+    };
+  }
+
+  async markAsExpired(challengeId) {
+    if (!await this.challengesWrapper.isInProgress(challengeId)) {
+      throw new Error(`Challenge ${challengeId} not found`);
+    }
+    if (!await this.challengesWrapper.challengeIsTimedOut(challengeId)) {
+      throw new Error(`Challenge ${challengeId} cannot be marked as expired yet`);
```
[the remark this answers is not in the document]
Comment (vanruch) [reply src/actions/challenges_actions.js]: 👍

File: src/tasks/challenges.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    }
+    console.log(`Challenge with ID = ${challengeId} is in progress.`);
+    if (status.canResolve) {
+      console.log('And it can be resolved by you.');
+    } else {
+      console.log('However, it cannot be resolved by you at the moment.');
+    }
+    if (status.isTimedOut) {
+      console.log(`This challenge is past its expiration date and nobody has resolved it :(. 
+Mark it as expired to punish the shelterer.`);
```
[the remark this answers is not in the document]
Comment (vanruch) [reply src/tasks/challenges.js]: If it is in progress and timed out then yes. Will fix wording

File: test/wrappers/challenges_wrapper.js (hunk 2 above)
Comment (vanruch) [reply test/wrappers/challenges_wrapper.js]: 👍

File: src/tasks/challenges.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (krzysztof-jelski) [reply src/tasks/challenges.js]: OK, then let's at least extract a method than names the condition clearly: `is64DigitsHexString`
